storage/innobase/xtrabackup/test/t/innodb_keyring_file_PXB_2275.sh

```diff
@@ -0,0 +1,111 @@
+. inc/common.sh
+. inc/keyring_file.sh
+
+if ! $XB_BIN --help 2>&1 | grep -q debug-sync; then
+    skip_test "Requires --debug-sync support"
```

Comment:
Please use smaller case letters for filename.
Also innodb is redundant imo.. how about keyring_pxb_2275.sh